internal/storage/storage_handle_test.go

```diff
@@ -298,3 +298,13 @@ func (testSuite *StorageHandleTest) TestCreateClientOptionForGRPCClient() {
 	assert.Nil(testSuite.T(), err)
 	assert.NotNil(testSuite.T(), clientOption)
 }
+
+func (testSuite *StorageHandleTest) TestNewStorageHandleWithMaxRetryAttemptsNotZero() {
+	sc := storageutil.GetDefaultStorageClientConfig()
+	sc.MaxRetryAttempts = 100
+
+	handleCreated, err := NewStorageHandle(context.Background(), sc)
+
+	assert.Nil(testSuite.T(), err)
+	assert.NotNil(testSuite.T(), handleCreated)
```

Comment:
You should only check whether handleCreated is not nil if there is no error.
```suggestion
	if assert.Nil(testSuite.T(), err) {
	    assert.NotNil(testSuite.T(), handleCreated)
	}
```

Reply:
Done